gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FilesService.java

```diff
@@ -39,15 +42,14 @@ public FilesService(NetworkService networkImporter, MergingViewService mergingVi
     }
 
     public SweData importFiles(SweRequest sweRequest) {
-
         MergingViewData mergingViewData = mergingViewService.importMergingView(sweRequest);
         Network networkEsFr = networkService.importNetwork(sweRequest);
         OffsetDateTime targetProcessDateTime = sweRequest.getTargetProcessDateTime();
         Network networkFrEs = networkService.loadNetworkFromMinio(targetProcessDateTime);
         Network networkEsPt = networkService.loadNetworkFromMinio(targetProcessDateTime);
         Network networkPtEs = networkService.loadNetworkFromMinio(targetProcessDateTime);
-        CimCracCreationContext cracCreationContextFrEs = fileImporter.importCracFromCimCracAndNetwork(fileImporter.importCimCrac(sweRequest), targetProcessDateTime, networkEsFr, CRAC_CIM_CRAC_CREATION_PARAMETERS_FR_ES_JSON);
-        CimCracCreationContext cracCreationContextEsPt = fileImporter.importCracFromCimCracAndNetwork(fileImporter.importCimCrac(sweRequest), targetProcessDateTime, networkEsPt, CRAC_CIM_CRAC_CREATION_PARAMETERS_PT_ES_JSON);
+        CimCracCreationContext cracCreationContextFrEs = fileImporter.importCracFromCimCracAndNetwork(fileImporter.importCimCrac(sweRequest), targetProcessDateTime, networkEsFr, sweRequest.getProcessType().equals(ProcessType.D2CC) ? CRAC_CIM_CRAC_CREATION_PARAMETERS_FR_ES_D2CC_JSON : CRAC_CIM_CRAC_CREATION_PARAMETERS_FR_ES_IDCC_JSON);
```

Comment:
For readability reasons, I would prefer this ternary operation to be stored in a variable rather than inlined here.